src/ray/gcs/pb_util.h

```diff
@@ -138,22 +138,6 @@ inline const rpc::RayException *GetCreationTaskExceptionFromDeathCause(
   return &(death_cause->creation_task_failure_context());
 }
 
-/// Generate object error type from ActorDeathCause.
```

Comment:
Why we have this in the first place?

Reply:
Hmm I think the error_type didn't exist before. It looks like it is not properly set everywhere?

Reply:
Where are these? I guess we never have tests for it also right...